src/coordinator.rs

```diff
@@ -3,16 +3,23 @@ use anyhow::{Context, Result};
 use roc_std::{RocList, RocStr};
 use std::collections::{HashMap, HashSet};
 use std::hash::{Hash, Hasher};
+use std::process::Command;
 
 #[derive(Debug, Eq, Hash, PartialEq, Clone, Copy)]
-struct JobId(u64);
+pub struct JobId(u64);
 
 impl From<u64> for JobId {
     fn from(unwrapped: u64) -> Self {
         JobId(unwrapped)
     }
 }
 
+impl std::fmt::Display for JobId {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{:x}", self.0)
```

Comment:
nit: why hex?

Reply:
it produces slightly shorter strings and I find hex easier to work with than digits alone for some reason (e.g. when debugging I often say "ok this is e-7-whatever and it appears here and here")

Reply:
Fair enough